cmd/main.go

```diff
@@ -145,6 +147,13 @@ func main() {
 
 	var openshiftVersionsMap models.OpenshiftVersions
 
+	if !isJSONString(Options.OpenshiftVersions) {
```

Comment:
Why not just verify whether it's empty

Reply:
what is wrong with validating it's a valid JSON?  
in theory, it can be `not empty` but still not a valid JSON

Reply:
I think in case of a wrong Json values - The pod shouldnt keep running but fail. It means someone tried to config something, wrongly,  but you just ignore it and use the defaults, without any logs or any warnings to the deployment initiator.